packages/grid/src/vaadin-grid-column-group.js

```diff
@@ -90,7 +90,6 @@ class GridColumnGroup extends ColumnBaseMixin(PolymerElement) {
       '_groupFrozenChanged(frozen, _rootColumns)',
       '_groupFrozenToEndChanged(frozenToEnd, _rootColumns)',
       '_groupHiddenChanged(hidden, _rootColumns)',
-      '_visibleChildColumnsChanged(_visibleChildColumns)',
```

Comment:
Removed this observer, and inlined the code of `_visibleChildColumnsChanged` into `_updateVisibleChildColumns`.